llm/gpt-2/README.md

```diff
@@ -0,0 +1,58 @@
+# Run GPT-2 in llm.c on any cloud with SkyPilot
+
+This is a reproducible package of llm.c's GPT-2 (124M) training by @karpathy (https://github.com/karpathy/llm.c/discussions/481)
+With SkyPilot, you can run GPT-2 (124M) training on any cloud.
+
+## Prerequisites
+
+1. Install [SkyPilot](https://github.com/skypilot-org/skypilot):
+```bash
+pip install skypilot-nightly
+```
+2. Enable clouds for SkyPilot:
+```bash
+sky check
+```
+Please check the instructions for enabling clouds at [SkyPilot doc](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).
+3. Download the YAMLs in this directory for data processing and training:
+```bash
+wget https://github.com/skypilot-org/skypilot/blob/master/llm/gpt-2/gpt2-data.yaml
+wget https://github.com/skypilot-org/skypilot/blob/master/llm/gpt-2/gpt2-train.yaml
+```
+
+## Data processing
```

Comment:
nit: I like the fact that we have different YAMLs for pre-processing and training, but one concern is it may alienate lambda-only, azure-only or fluidstack-only users since they won't have any cloud object store access to write preprocessed data to. 

If it's not too complicated, can we add a one-shot YAML that does all pre-processing and training in a single YAML?